docs/ref/unicode.txt

```diff
@@ -35,58 +36,36 @@ able to store certain characters in the database, and information will be lost.
 .. _section 2: https://docs.oracle.com/database/121/NLSPG/ch2charset.htm#NLSPG002
 .. _section 11: https://docs.oracle.com/database/121/NLSPG/ch11charsetmig.htm#NLSPG011
 
-All of Django's database backends automatically convert strings into
+All of Django's database backends automatically convert text into bytes using
 the appropriate encoding for talking to the database. They also automatically
-convert strings retrieved from the database into strings. You don't even need
+convert bytes retrieved from the database into text. You don't even need
 to tell Django what encoding your database uses: that is handled transparently.
 
 For more, see the section "The database API" below.
 
 General string handling
 =======================
 
-Whenever you use strings with Django -- e.g., in database lookups, template
-rendering or anywhere else -- you have two choices for encoding those strings.
-You can use normal strings or bytestrings (starting with a 'b').
+Whenever you use strings with Django -- e.g.,
+in database lookups, template rendering or anywhere else -- you can use text
+or bytes.
```

Comment:
I'm not sure if this will still be the case considering that `force_text() usage [will be limited](https://code.djangoproject.com/ticket/27795).